viewflow/lock.py

```diff
@@ -1,16 +1,34 @@
 """
-Prevents unconsistent db updates for flow.
+Prevents inconsistent db updates for flow.
 """
 import time
 import random
 from contextlib import contextmanager
 
-from django.core.cache import cache
+from django.core.cache import DEFAULT_CACHE_ALIAS
 from django.db import transaction, DatabaseError
+from django.conf import settings
 
 from viewflow.exceptions import FlowLockFailed
 
 
+CACHE_BACKEND = getattr(settings, 'VIEWFLOW_LOCK_CACHE_BACKEND', DEFAULT_CACHE_ALIAS)
```

Comment:
I don't want to introduce any viewflow related settings at this time.

Generally, in order not to get into settings hell, I suppose to design settings as a single `VIEWFLOW` settings dictionary (ex: django TEMPLATES). And all settings access in the viewflow library will goes from `viewflow.settings` module, that will responsible for settings validation and default values.